src/main/java/org/openrewrite/java/testing/assertj/SimplifyAssertJAssertion.java

```diff
@@ -95,11 +95,11 @@ public J.MethodInvocation visitMethodInvocation(J.MethodInvocation methodInvocat
             }
 
             // Assume zero argument replacement method
-            return JavaTemplate.builder(dedicatedAssertion + "()")
-                    .contextSensitive()
+            String receiverType = mi.getSelect().getType().toString().replaceAll("<.*>", "");
+            return JavaTemplate.builder("#{any(" + receiverType + ")}." + dedicatedAssertion + "()")
```

Comment:
I would instead try with this:
```suggestion
            return JavaTemplate.builder("#{any()}." + dedicatedAssertion + "()")
```
This should work (while keeping the `apply()` call as-is), since it should then retrieve the type from the method's select.